map/framework.cpp

```diff
@@ -1978,7 +1978,8 @@ void Framework::ActivateMapSelection()
                                 bi.m_needAnimationOnSelection, bi.m_isGeometrySelectionAllowed, true);
   }
 
-  ASSERT(m_onPlacePageOpen, ());
+  /// @todo Current android logic is strange (see SetPlacePageListeners comments), so skip assert.
```

Comment:
This doesn't explain why this assert started to fail recently. Monkey has started to failed  4-5 days ago.

Reply:
? Bad commit was merged 6 days ago.

Reply:
Ah, I see. OK!